v1.1/uuid.md

```diff
@@ -0,0 +1,135 @@
+---
+title: UUID
+summary: The UUID data type .
+toc: false
+---
+
+The `UUID` (Universally Unique Identifier) [data type](data-types.html) stores a 128-bit value that is [unique across both space and time](https://www.ietf.org/rfc/rfc4122.txt). 
```

Comment:
Let's add the ["New in 1.1" callout](https://github.com/cockroachdb/docs/blob/master/CONTRIBUTING.md#version-tags) to the beginning of this paragraph.